tests/test_shell.py

```diff
@@ -86,3 +86,33 @@ def test_init_flowproject_template(self, template):
         self.call(f"python -m flow init -t {template}".split())
         assert str(signac.get_project()) == "project"
         assert os.path.exists("project.py")
+
+    def test_template_create_base(self):
+        self.call("python -m flow init".split())
+        self.call("python -m flow template create".split())
+        assert os.path.exists("templates/script.sh")
+        template = flow.environment.get_environment().template
+        with open("templates/script.sh") as fh:
+            custom_script_lines = fh.read().splitlines()
+        assert ('{% extends "' + template + '" %}') in custom_script_lines[0]
+
+    def test_template_create_fail_on_recall(self):
+        self.call("python -m flow init".split())
+        self.call("python -m flow template create".split())
+        with pytest.raises(ExitCodeError):
+            self.call("python -m flow template create".split())
+
+    @pytest.mark.parametrize("name_arg", ("-n foo.sh", "--name foo.sh"))
+    def test_template_create_custom_name(self, name_arg):
+        self.call("python -m flow init".split())
+        self.call(("python -m flow template create " + name_arg).split())
+        assert os.path.exists("templates/foo.sh")
```

Comment:
Verify the file contents. Something like what's above:
```suggestion
        assert os.path.exists("templates/foo.sh")
        with open("templates/foo.sh") as fh:
            custom_script_lines = fh.read().splitlines()
        assert f'{% extends "{template}" %}' in custom_script_lines[0]
```